### DISCUSSION TITLE
Can't access LS420D

### DISCUSSION THREAD

**Discussion by kf4rtg** [1000001101000/Debian_on_Buffalo#221] [Q&A] [answered]
Very excited about this project. The existing software was awful and my NAS sat in my junk box for awhile until I came across this project. I used the debootstrap script and it seems like I got through everything. Now I have booted up the NAS and when I try to connect with Putty I get an immediate Network Error: Connection refused. When I try to open up the IP in a browser I get 200 OK, and nothing more. 
Is there a different port I should be using with SSH or the web browser to get access and finish the configuration?

**Comment by 1000001101000** [comment, marked as answer]
Sounds like you’ve got the wrong IP or not running debian, debian would be accepting ssh and wouldn’t respond to http requests

**Reply by kf4rtg** [reply]
You are RIGHT. What an awesome first post for me ::facepalm:: 
That was the dimmer I installed in the den last weekend. the Hostname was KS420 which I thought I made a typo when I added the hostname to the script LS240

**Comment by 1000001101000** [comment]
No worries. It's pretty common for the device to end up with a new IP which confuses people when they don't get a response from the old IP. It's entirely possible for something else to come along and be issued the old IP depending on the setup but I don't think I've heard from anyone with that happening before.

**Reply by kf4rtg** [reply]
Yeah- Hadn't even played with the nas since I realized a need for it several months back (linux gameserver backups) fired it up, had to rebuild array and update firmware, then shelved it because it was slow and lacked features. I used an IP scan to find it and jumped on the first thing that looked plausible. Good to know that my smart dimmer has a webserver I guess....
